content/python/concepts/built-in-functions/terms/next/next.md

```diff
@@ -0,0 +1,71 @@
+---
+Title: 'next()'
+Description: 'Returns the next element from an iterable object.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Collections'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+The **`next()`** function returns the next element from an [iterator](https://www.codecademy.com/resources/docs/python/iterators) object.
+
+## Syntax
+
+```pseudo
+next(iterator_name, [default_parameter])
+```
+The required `iterator_name` parameter is the iterator object with the set of values. The `default_parameter` is optional and is printed if the end of the iterator is reached.
```

Comment:
```suggestion

The required `iterator_name` parameter is the iterator object with the set of values. The `default_parameter` is optional and is printed if the end of the iterator is reached.
```

Reply:
I can´t see the difference between line 23 and line 24.

Reply:
¿The colour of and & if? What for?

Reply:
I have just updated the file with your suggestions. Thanks.